frontend/lib/src/components/elements/ChatMessage/ChatMessage.tsx

```diff
@@ -58,7 +60,7 @@ function ChatMessageAvatar(props: ChatMessageAvatarProps): ReactElement {
         if (avatar === "user") {
           return (
             <StyledAvatarIcon
-              data-testid="chatAvatarIcon-user"
+              data-testid="stChatMessageAvatarUser"
```

Comment:
should they also have a `className` attribute or they don't because they are never used as top-level components?

Reply:
Yep, for this project we only want to guarantee a stable class name on the top-level div of elements.